client.go

```diff
@@ -678,31 +714,45 @@ func (c *Client) runRepliesConsumer(inChan *amqp.Channel) error {
 	}
 
 	go func() {
-		c.debugLog("client: running replies consumer...")
+		c.logger.Debug("client: running replies consumer...")
 
 		for response := range messages {
 			request, ok := c.requestsMap.GetByCorrelationID(response.CorrelationId)
 			if !ok {
-				c.errorLog(
-					"client: could not find where to reply. CorrelationId: %s",
-					stringifyDeliveryForLog(&response),
+				// The request has probably timed out during the it was in the
+				// queue and the user has stopped waiting for it. We can safely
```

Comment:
This sentence seems odd, maybe something like

```suggestion
				// The request has probably timed out between when it was in the
				// queue and when the user stopped waiting for it. We can safely
```